### DISCUSSION TITLE
New LoRa Model for XIAO

### DISCUSSION THREAD

**Discussion by kevin-seeed** [Seeed-Studio/OSHW-XIAO-Series#15] [Accomplished]
XIAO has never had a dedicated LoRa extension board. Now, we have developed a LoRa module based on the Semtech SX1262 chip, and used this module to create an extension board for XIAO. This board communicates with XIAO via SPI, allowing XIAO nRF52840 or XIAO ESP32 to develop applications based on LoRa or LoRaWAN. Please share your thoughts with us, so we can quickly design a module or extension board that meets everyone's needs.
![Wio-SX1262](https://github.com/Seeed-Studio/XIAO_Series/assets/170942971/dd990481-1df4-45e3-af62-c3a3e7566c32)
![DSC01075](https://github.com/Seeed-Studio/XIAO_Series/assets/170942971/c279ce8a-85d5-467f-8d6c-3fb8da677f3d)

**Comment by kkwestt** [comment]
xiao meshtastic ??? good good

**Reply by Uspizig** [reply]
very nice idea

**Comment by meilily** [comment]
Hey community, as this proposal went so popular, we've now already made a dream come true. Meet [XIAO ESP32S3 for Meshtastic & LoRa](https://www.seeedstudio.com/Wio-SX1262-with-XIAO-ESP32S3-p-5982.html). This thumb-sized LoRa dev kit, featuring the XIAO ESP32S3 and [Wio-SX1262 LoRa module](https://www.seeedstudio.com/Wio-SX1262-Wireless-Module-p-5981.html), is an ideal starter kit for Meshtastic and LoRa/LoRaWAN projects.
